### DISCUSSION TITLE
Data Gathering

### DISCUSSION THREAD

**Discussion by MStahl15** [GoogleChrome/lighthouse#12886] [General]
Greetings!  

The company I work for deals with PHI/PII (Protected Health Information/Personally Identifiable Information) and I'm curious as to the data that Lighthouse gathers on its end, if any?  

For obvious reasons, dealing with peoples PHI/PII, this information cannot be exposed or gathered by un-approved or un-contracted third parties so I'm hesitant on implementing Lighthouse into the applicable repos. 

Any insight to this will be much appreciated!!

**Comment by connorjclark** [comment]
If you're running Lighthouse yourself locally (CLI, DevTools, ... NOT the chrome extension), you'll be happy to know we collect nothing except for the opt-in Sentry gathering in the CLI (which will only send us stack traces and URLs).

**Reply by MStahl15** [reply]
What about runs that are not local?  Looking to have this integrated in our CircleCI pipeline running against multiple environments, including production via a CRON job.  

If the answer is the same, then awesome - I'll move forward with implementation.  If not, would you mind explaining what exactly is gathered in this scenario, or point me to the proper documentation?

**Reply by brendankenny** [reply]
If you're using the CLI or node module, those count as local runs too (just local to the CI machine) and Lighthouse logs no information by default. A non-local run would be something like [PSI](https://developers.google.com/speed/pagespeed/insights/), where Lighthouse is hosted and run for you and is covered by the usual Google privacy policy (or whoever is hosting the service if it's somewhere else).

**Reply by brendankenny** [reply]
docs for the error logging @connorjclark mentioned: https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md

but as he said, it's opt-in for the CLI only, and there won't even be an opportunity to opt-in because the prompt is disabled when `process.env.CI` is set to `true` (as [it is in CircleCI](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables))
